RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -411,29 +426,44 @@ public void setCurrentFile(String currentFile) {
     }
 
     @Override
-    public void onPause(){
+    public void onPause() {
         super.onPause();
         dismissExportProgressDialog();
-        dismissDatabaseResyncProgressDialog();
     }
 
-    private void dismissExportProgressDialog(){
-        if(mPd != null && mPd.isShowing()){
+    private void dismissExportProgressDialog() {
```

Comment:
The name tells me that this function only dismiss a progress dialog if it's of the "Export" type. But the function actually closes the progress dialog agnostic to the type (zipping/export).